terraform/README.md

```diff
@@ -1,18 +1,35 @@
-# Terraform modules for Digital Marketplace
+# Terraform for the Digital Marketplace
 
-## Root modules
+## Projects and modules
 
- * **aws-dm**: digitalmarketplace AWS account
- * **aws-dm-dev**: digitalmarketplace-development AWS account
- * **aws-dm-prod**: digitalmarketplace-production AWS account
+The following projects each define and control the state of a section of our infrastructure.
+
+ * **accounts/development**: The digitalmarketplace-development AWS account
+ * **accounts/main**: The digitalmarketplace AWS account
+ * **accounts/production**: The digitalmarketplace-production AWS account
+ * **environments/preview**
+ * **environments/staging**
+ * **environments/production**
+
+Account projects apply to the AWS account as a whole, so should only contain resources that are created once per
+account (eg root Route53 hosted zones, IAM users).
+
+Environment projects apply to individual environments and should contain resources that should be created for each
+environment (eg application logs and S3 buckets).
+
+All projects are then able to import and use the functionality they require from our folder of shared modules.
 
 ## Requirements
 
+You will need to install the following manually (they are not installed as part of any other process such as
+`make requirements` or `pip`).
+
 ### Install Terraform
 
-Make sure you have at least v0.8 installed: https://www.terraform.io/downloads.html
+Make sure you have at least v0.11.2 installed: https://www.terraform.io/downloads.html
```

Comment:
I think this is tricky. When George and I were playing around with Terraform we noticed that it reported it was backwards incompatible (i.e. if someone has run terraform plan/apply with version v0.11.3 and I come along and try to run it with v0.11.2, it will say that I need to update my tool). This might mean that suggesting a version number here is pretty much always going to be out of date as Terraform forces everyone onto the latest version.

Reply:
I've added a paragraph to clarify this.